neat/trees/compartmenttree.py

```diff
@@ -341,10 +339,7 @@ def getDynamicDrive(self, channel_name, p_open, v):
     def getDynamicDrive_(self, channel_name, v, dt, channel_storage=None):
         # assert p_open.shape == v.shape
         _, e = self.currents[channel_name]
-        if channel_storage is not None:
```

Comment:
not sure these changes are on purpose as they don't seem related to fixing reqs?

Reply:
Came across this deprecated part of the code while checking whether some import statements could be removed.